app.js

```diff
@@ -0,0 +1,34 @@
+
+const { request, response } = require('express')
+const express = require('express')
+
+//Init Express
+const app = express()
+
+//2. End points/routes handlers
+
+//make everything inside of public/available
+app.use(express.static('public'))
+
+//our first Route:
+app.get('/home', (request, response, next) => {
```

Comment:
Aquí podríamos dejar solo la ruta raíz ("/")para que lo primero que cargue sea tu home 
```suggestion
app.get('/', (request, response, next) => {
```